### ISSUE TITLE
UI Design

### ISSUE THREAD

**Issue by UO287621** [Arquisoft/wiq_en1a#46]
We should be discussing the design of the webpage. As you might have noticed, I have included **_Tailwind_** in the project (a CSS Framework), that helps styling components faster. 

We should agree on a **color palette**; for this task I usually use [ColorHunt](https://colorhunt.co/)

I remember @andrrsin was interested in the aesthetics design of the application so he might be interested in this issue

**Comment by andrrsin** [comment]
Perhaps we can try to use a purple/orange palette like retrowave. And we could try to reach that style as much as we can with our knowledge. As they are colors that even just being solid can be good :)

**Comment by UO287621** [comment]
Currently i have designed a basic minimalistic white website. We will add color and make it more vivid for the following prototype maybe?

**Comment by ChristianFN2** [comment]
I agree, for now let's fix the login and have the game working without bugs

**Comment by adriiglz** [comment]
Moving this issue to a discussion as this is not really an "issue"
